### ISSUE TITLE
Pretrained model for projection_layer

### ISSUE THREAD

**Issue by jm20041050** [bcmi/PHDiffusion-Painterly-Image-Harmonization#5]
In the `loss/VGG_Loss.py` file, there is `projection_style` which is used for the contrative style loss.
However, in the code, there is no code to load state_dict and pretrained model path.

How can I get the information of `projection_style`?

**Comment by pokaaa** [comment]
The projection layer can be trained along with the entire network by including its parameters in the optimizer. The usage of this two-layer small network is based on this paper(git@github.com:HalbertCH/IEContraAST.git), and you can refer to it for more information related to it.
